notebooks/signac_107_HOOMD-blue.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>We need to either update to the new hoomd sytnax when it hits conda-forge (if it hasn't already?) or pin hoomd to the 2.x series</p><pre class="ql-syntax" spellcheck="false">conda install -c conda-forge hoomd=2 numpy signac signac-flow 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
The `environment.yml` is already pinned to `hoomd=2` and HOOMD 3 likely won't be finalized for several months. For now, we're sticking to HOOMD v2 and deferring a rewrite of the tutorials using HOOMD. See related issue #18.